website/src/categories/components/breadcrumbs/usage.md

```diff
@@ -5,27 +5,129 @@ eleventyNavigation:
   parent: Breadcrumbs
   key: CardUsage
 ---
+<section class="no-heading">
+
+<div class="ds-example">
+<sl-breadcrumbs homeUrl="/">
+  <a href="javascript:void(0)">Settings</a>
+  <a href="javascript:void(0)">My profile</a>
+  <a href="javascript:void(0)">Details</a>
+</sl-breadcrumbs>
+</div>
+
+<div class="ds-code">
+
+  ```html
+    <sl-breadcrumbs homeUrl="/">
```

Comment:
And here; isn't `/` the default home url???

Reply:
right :)